modules/gsuite_enabled/variables.tf

```diff
@@ -152,7 +152,7 @@ variable "disable_services_on_destroy" {
 }
 
 variable "default_service_account" {
-  description = "Project default service account setting: (delete | depriviledge | keep)"
+  description = "Project default service account setting: delete; depriviledge; keep."
```

Comment:
Thoughts on spelling this out, e.g. "Project default service account setting. Can be one of `delete`, `depriviledge`, or `keep`."

Reply:
```suggestion
  description = "Project default service account setting: can be one of `delete, `depriviledge`, or `keep`."
```